src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Packaging Theia as a Desktop Product
+---
+
+# Packaging Theia as a Desktop Product
+
+Eclipse Theia Blueprint product is a template to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform.
+Theia Blueprint assembles a selected subset of existing Theia features and extensions.
+We provide installers for Theia Blueprint to be downloaded (see below) from the Theia webpage.
```

Comment:
Rather than "(see below)" it'd be better to link the section.

Reply:
These kind of links don't work with the current gatsby plugins as far as I could see.